system/lib/mimalloc/include/mimalloc.h

```diff
@@ -8,7 +8,7 @@ terms of the MIT license. A copy of the license can be found in the file
 #ifndef MIMALLOC_H
 #define MIMALLOC_H
 
-#define MI_MALLOC_VERSION 217   // major + 2 digits minor
+#define MI_MALLOC_VERSION 188   // major + 2 digits minor
```

Comment:
Ah, I just noticed that I synced with the `dev` branch. Perhaps it should be synced with the `dev-slice` branch instead.
https://github.com/emscripten-core/emscripten/blob/8d59f19ef9aeeabeb23f1977d7ccd67fbd3fe215/system/lib/mimalloc/readme.md?plain=1#L71-L76